src/app/components/login/reset-password-dialog/reset-password-dialog.component.html

```diff
@@ -0,0 +1,32 @@
+<div *ngIf="!data.emailSent; else elseBlock">
```

Comment:
Jeg synes at der skal arbejdes lidt mere med designet. 
I retning ala https://proxy.duckduckgo.com/iu/?u=http%3A%2F%2Fimage.shutterstock.com%2Fz%2Fstock-vector-ui-elements-login-registration-and-forgot-password-form-flat-design-261288158.jpg&f=1

Du kan google "material design forgot password"  og få inspiration fra billeder der